packages/reactivity/src/ref.ts

```diff
@@ -85,10 +110,50 @@ declare const ShallowRefMarker: unique symbol
 
 export type ShallowRef<T = any> = Ref<T> & { [ShallowRefMarker]?: true }
 
+/**
+ * Creates a ref that tracks mutations on its own `value` but doesn't make its
```

Comment:
I think using the version of the api docs should be better here https://vuejs.org/api/reactivity-advanced.html#shallowref